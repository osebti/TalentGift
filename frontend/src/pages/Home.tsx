// components
import Button from "@/components/Button";
import NavBar from "@/components/NavBar";

interface Step {
  title: string;
  content: string;
  color: string;
}

interface Reason {
  title: string;
  content: string;
  imgSource: string;
  color: string;
}

export default function Home() {
  const steps: Step[] = [
    {
      title: "Create an Organization",
      content:
        "Empower your team to navigate corporate challenges.",
      color: "bg-lightPurple",
    },
    {
      title: "Take a Survey",
      content: "Share your thoughts and make your voice count!",
      color: "bg-darkPurple",
    },
    {
      title: "Receive Feedback",
      content:
        "Keep your team engaged with personalized notifications and insights.",
      color: "bg-green",
    },
  ];

  const reasons: Reason[] = [
    {
      title: "Employee Engagement",
      content:
        "Open up the channels and lend an ear to your incredible team. By actively listening to your employees, you're not just making decisionsâ€”you're orchestrating a symphony of collaboration, paving the way for meaningful change within your organization.",
      imgSource: "/engineer_team.svg",
      color: "bg-lightPurple",
    },
    {
      title: "Performance Management",
      content:
        "Elevate your teams to new heights by fostering ongoing conversations. From anytime feedback and goal tracking to development-focused reports and enriched 1-on-1s, we empower you to build high-performing teams that thrive.",
      imgSource: "/pie_graph.svg",
      color: "bg-darkPurple",
    },
    {
      title: "Employee Development",
      content:
        "Nurture and keep your talent flourishing through personalized, continuous development that's not only easy to measure but also scalable for sustained growth.",
      imgSource: "/report.svg",
      color: "bg-green",
    },
  ];

  return (
    <div className="flex flex-col gap-y-16 mt-4">
      <NavBar hideButton={false} className="mx-4 md:mx-8" />
      {/* Hero Section */}
      <section className="relative grid grid-cols-1 sm:gap-x-20 sm:grid-cols-2 place-items-center px-4 md:px-8 md:py-20">
        <div className="absolute w-full h-full bg-lightPurple opacity-20 z-[-1]"></div>
        <div className="flex flex-col gap-y-8">
          <h1 className="font-semibold text-center md:text-start text-2xl md:text-3xl lg:text-6xl text-black opacity-90">
            Thrive Together for a Vibrant Workplace Culture
          </h1>
          <p className="text-sm md:text-base text-center md:text-start">
            Equip your organization with essential tools and insights for
            employee engagement, performance, and growth.
          </p>
          <div className="flex justify-between mx-8 md:mx-0 gap-x-8">
            <Button
              id="join-now"
              label="Join Now"
              href="/sign-up"
              navigate
              style="btn-secondary flex-grow"
            />
            <Button
              id="our-goals"
              label="Our goals"
              style="btn-accent flex-grow"
              onClick={() => {
                const elem: HTMLElement | null =
                  document.getElementById("goals");
                elem?.scrollIntoView({ behavior: "smooth" });
              }}
            />
          </div>
        </div>
        <div className="relative min-h-[20rem] w-full h-full col-start-2 flex justify-center my-[8%]">
          <img
            alt="Mindset"
            src="/mindset.svg"
            className="object-contain h-full absolute"
          />
        </div>
      </section>
      {/* How does it work? */}
      <section className="px-4 md:px-8 flex flex-col gap-y-16">
        <h1 className="font-semibold text-2xl md:text-3xl lg:text-4xl">
          How does it work?
        </h1>
        <div className="flex flex-col lg:flex-row gap-y-4 md:gap-x-2 lg:gap-x-4">
          {steps.map((elem, index) => {
            return (
              <div
                key={index}
                className="card lg:w-1/3 min-h-[14rem] cursor-pointer group"
              >
                <div
                  className={`card-body relative text-white text-center lg:group-hover:translate-y-4 transition ease-in-out`}
                >
                  <div
                    className={`absolute ${elem.color} top-0 left-0 w-full h-full z-[-1] opacity-[0.90]`}
                  ></div>
                  <h2 className="font-semibold text-lg md:text-2xl whitespace-nowrap">
                    {elem.title}
                  </h2>
                  <p className="text-sm md:text-base mt-[8%]">
                    {elem.content}
                  </p>
                </div>
              </div>
            );
          })}
        </div>
      </section>
      {/* Why Us? */}
      <section id="goals" className="px-4 md:px-8 flex flex-col gap-y-8">
        <h1 className="font-semibold text-2xl md:text-3xl lg:text-4xl pt-4 md:pt-8">
          Why Us?
        </h1>
        {/* desktop layout */}
        {reasons.map((elem, index) => {
          return (
            <div
              key={index}
              className="md:grid md:grid-cols-2 grid-rows-1 place-items-center gap-x-8 group hidden"
            >
              {index % 2 == 1 ? (
                <div className="relative min-h-[20rem] w-full h-full flex justify-center my-[4%]">
                  <img
                    alt="Illustrations"
                    src={elem.imgSource}
                    className="object-contain h-full absolute"
                  ></img>
                </div>
              ) : null}
              <div className="flex flex-col gap-y-4">
                <h2 className="text-darkPurple font-semibold text-2xl md:text-3xl group-hover:text-lightPurple transition ease-in-out">
                  {elem.title}
                </h2>
                <p className="text-base md:text-lg">{elem.content}</p>
              </div>
              {index % 2 == 0 ? (
                <div className="relative min-h-[20rem] w-full h-full flex justify-center my-[4%]">
                  <img
                    alt="illustrations"
                    src={elem.imgSource}
                    className="object-contain h-full absolute"
                  ></img>
                </div>
              ) : null}
            </div>
          );
        })}
        {/* mobile layout */}
        <div className="flex flex-col gap-y-4 md:hidden">
          {reasons.map((elem, index) => {
            return (
              <div key={index} className="card min-h-[14rem] cursor-pointer">
                <div className={`card-body relative text-white text-center`}>
                  <div
                    className={`absolute ${elem.color} top-0 left-0 w-full h-full z-[-1] opacity-[0.90]`}
                  ></div>
                  <h2 className="font-semibold text-lg whitespace-nowrap">
                    {elem.title}
                  </h2>
                  <p className="text-base mt-[8%]">{elem.content}</p>
                </div>
              </div>
            );
          })}
        </div>
      </section>
    </div>
  );
}
