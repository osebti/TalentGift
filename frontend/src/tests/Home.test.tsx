import { cleanup, fireEvent, render } from "@testing-library/react";
import "@testing-library/jest-dom";

import Home from "@/pages/Home";
import { BrowserRouter } from "react-router-dom";

afterEach(cleanup);

function renderHomePage() {
  return (
    render(
      <BrowserRouter>
        <Home/>
      </BrowserRouter>
    )
  );
}

describe("Rendering the Landing Page", () => {

  it("renders correctly", () => {
    const page = renderHomePage();
    expect(page).toMatchSnapshot();
  });
  
  it("renders sign in button", () => {
    const { queryByTestId } = renderHomePage();
    expect(queryByTestId("Sign In-button")).not.toBeNull();
  });

  it("renders sign up button", () => {
    const { queryByTestId } = renderHomePage();
    expect(queryByTestId("Sign Up-button")).not.toBeNull();
  });

});

describe("Landing Page Navigation", () => {
  
  it("directs to sign in page when sign in button is clicked", () => {
    const { queryByText } = renderHomePage();
    // click on sign in button
    const signInButton = queryByText("Sign In");
    if (signInButton != null) {
      fireEvent.click(signInButton);
    }
    // check if site navigates to correct address
    expect(global.window.location.href).toContain("/sign-in");
  });

  it("directs to sign up page when sign up button is clicked", () => {
    const { queryByText } = renderHomePage();
    // click on sign up button
    const signUpButton = queryByText("Sign Up");
    if (signUpButton != null) {
      fireEvent.click(signUpButton);
    }
    // check if site navigates to correct address
    expect(global.window.location.href).toContain("/sign-up");
  });
  
});
