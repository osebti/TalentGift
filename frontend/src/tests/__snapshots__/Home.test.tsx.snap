// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering the Landing Page renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex flex-col gap-y-16 mt-4"
      >
        <div
          class="mx-4 md:mx-8 flex justify-between items-center"
        >
          <a
            class="font-[1000] text-darkPurple text-3xl bg-transparent border-0"
            href="/"
          >
            TG
          </a>
          <div
            class="flex gap-x-4"
          >
            <a
              class="btn btn-md rounded-full capitalize font-normal text-white text-sm md:text-base btn-accent btn-sm md:btn-md px-4 md:px-8"
              data-testid="Sign In-button"
              href="/sign-in"
              id="Sign In-button"
            >
              Sign In
            </a>
            <a
              class="btn btn-md rounded-full capitalize font-normal text-white text-sm md:text-base btn-primary btn-sm md:btn-md px-4 md:px-8"
              data-testid="Sign Up-button"
              href="/sign-up"
              id="Sign Up-button"
            >
              Sign Up
            </a>
          </div>
        </div>
        <section
          class="relative grid grid-cols-1 sm:gap-x-20 sm:grid-cols-2 place-items-center px-4 md:px-8 md:py-20"
        >
          <div
            class="absolute w-full h-full bg-lightPurple opacity-20 z-[-1]"
          />
          <div
            class="flex flex-col gap-y-8"
          >
            <h1
              class="font-semibold text-center md:text-start text-2xl md:text-3xl lg:text-6xl text-black opacity-90"
            >
              Thrive Together for a Vibrant Workplace Culture
            </h1>
            <p
              class="text-sm md:text-base text-center md:text-start"
            >
              Equip your organization with essential tools and insights for employee engagement, performance, and growth.
            </p>
            <div
              class="flex justify-between mx-8 md:mx-0 gap-x-8"
            >
              <a
                class="btn btn-md rounded-full capitalize font-normal text-white text-sm md:text-base btn-secondary flex-grow"
                data-testid="join-now-button"
                href="/sign-up"
                id="join-now-button"
              >
                Join Now
              </a>
              <button
                class="btn btn-md rounded-full capitalize font-normal text-white text-sm md:text-base btn-accent flex-grow"
                data-testid="our-goals-button"
                id="our-goals-button"
              >
                Our goals
              </button>
            </div>
          </div>
          <div
            class="relative min-h-[20rem] w-full h-full col-start-2 flex justify-center my-[8%]"
          >
            <img
              alt="Mindset"
              class="object-contain h-full absolute"
              src="/mindset.svg"
            />
          </div>
        </section>
        <section
          class="px-4 md:px-8 flex flex-col gap-y-16"
        >
          <h1
            class="font-semibold text-2xl md:text-3xl lg:text-4xl"
          >
            How does it work?
          </h1>
          <div
            class="flex flex-col lg:flex-row gap-y-4 md:gap-x-2 lg:gap-x-4"
          >
            <div
              class="card lg:w-1/3 min-h-[14rem] cursor-pointer group"
            >
              <div
                class="card-body relative text-white text-center lg:group-hover:translate-y-4 transition ease-in-out"
              >
                <div
                  class="absolute bg-lightPurple top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
                />
                <h2
                  class="font-semibold text-lg md:text-2xl whitespace-nowrap"
                >
                  Create an Organization
                </h2>
                <p
                  class="text-sm md:text-base mt-[8%]"
                >
                  Empower your team to navigate corporate challenges.
                </p>
              </div>
            </div>
            <div
              class="card lg:w-1/3 min-h-[14rem] cursor-pointer group"
            >
              <div
                class="card-body relative text-white text-center lg:group-hover:translate-y-4 transition ease-in-out"
              >
                <div
                  class="absolute bg-darkPurple top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
                />
                <h2
                  class="font-semibold text-lg md:text-2xl whitespace-nowrap"
                >
                  Take a Survey
                </h2>
                <p
                  class="text-sm md:text-base mt-[8%]"
                >
                  Share your thoughts and make your voice count!
                </p>
              </div>
            </div>
            <div
              class="card lg:w-1/3 min-h-[14rem] cursor-pointer group"
            >
              <div
                class="card-body relative text-white text-center lg:group-hover:translate-y-4 transition ease-in-out"
              >
                <div
                  class="absolute bg-green top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
                />
                <h2
                  class="font-semibold text-lg md:text-2xl whitespace-nowrap"
                >
                  Receive Feedback
                </h2>
                <p
                  class="text-sm md:text-base mt-[8%]"
                >
                  Keep your team engaged with personalized notifications and insights.
                </p>
              </div>
            </div>
          </div>
        </section>
        <section
          class="px-4 md:px-8 flex flex-col gap-y-8"
          id="goals"
        >
          <h1
            class="font-semibold text-2xl md:text-3xl lg:text-4xl pt-4 md:pt-8"
          >
            Why Us?
          </h1>
          <div
            class="md:grid md:grid-cols-2 grid-rows-1 place-items-center gap-x-8 group hidden"
          >
            <div
              class="flex flex-col gap-y-4"
            >
              <h2
                class="text-darkPurple font-semibold text-2xl md:text-3xl group-hover:text-lightPurple transition ease-in-out"
              >
                Employee Engagement
              </h2>
              <p
                class="text-base md:text-lg"
              >
                Open up the channels and lend an ear to your incredible team. By actively listening to your employees, you're not just making decisionsâ€”you're orchestrating a symphony of collaboration, paving the way for meaningful change within your organization.
              </p>
            </div>
            <div
              class="relative min-h-[20rem] w-full h-full flex justify-center my-[4%]"
            >
              <img
                alt="illustrations"
                class="object-contain h-full absolute"
                src="/engineer_team.svg"
              />
            </div>
          </div>
          <div
            class="md:grid md:grid-cols-2 grid-rows-1 place-items-center gap-x-8 group hidden"
          >
            <div
              class="relative min-h-[20rem] w-full h-full flex justify-center my-[4%]"
            >
              <img
                alt="Illustrations"
                class="object-contain h-full absolute"
                src="/pie_graph.svg"
              />
            </div>
            <div
              class="flex flex-col gap-y-4"
            >
              <h2
                class="text-darkPurple font-semibold text-2xl md:text-3xl group-hover:text-lightPurple transition ease-in-out"
              >
                Performance Management
              </h2>
              <p
                class="text-base md:text-lg"
              >
                Elevate your teams to new heights by fostering ongoing conversations. From anytime feedback and goal tracking to development-focused reports and enriched 1-on-1s, we empower you to build high-performing teams that thrive.
              </p>
            </div>
          </div>
          <div
            class="md:grid md:grid-cols-2 grid-rows-1 place-items-center gap-x-8 group hidden"
          >
            <div
              class="flex flex-col gap-y-4"
            >
              <h2
                class="text-darkPurple font-semibold text-2xl md:text-3xl group-hover:text-lightPurple transition ease-in-out"
              >
                Employee Development
              </h2>
              <p
                class="text-base md:text-lg"
              >
                Nurture and keep your talent flourishing through personalized, continuous development that's not only easy to measure but also scalable for sustained growth.
              </p>
            </div>
            <div
              class="relative min-h-[20rem] w-full h-full flex justify-center my-[4%]"
            >
              <img
                alt="illustrations"
                class="object-contain h-full absolute"
                src="/report.svg"
              />
            </div>
          </div>
          <div
            class="flex flex-col gap-y-4 md:hidden"
          >
            <div
              class="card min-h-[14rem] cursor-pointer"
            >
              <div
                class="card-body relative text-white text-center"
              >
                <div
                  class="absolute bg-lightPurple top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
                />
                <h2
                  class="font-semibold text-lg whitespace-nowrap"
                >
                  Employee Engagement
                </h2>
                <p
                  class="text-base mt-[8%]"
                >
                  Open up the channels and lend an ear to your incredible team. By actively listening to your employees, you're not just making decisionsâ€”you're orchestrating a symphony of collaboration, paving the way for meaningful change within your organization.
                </p>
              </div>
            </div>
            <div
              class="card min-h-[14rem] cursor-pointer"
            >
              <div
                class="card-body relative text-white text-center"
              >
                <div
                  class="absolute bg-darkPurple top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
                />
                <h2
                  class="font-semibold text-lg whitespace-nowrap"
                >
                  Performance Management
                </h2>
                <p
                  class="text-base mt-[8%]"
                >
                  Elevate your teams to new heights by fostering ongoing conversations. From anytime feedback and goal tracking to development-focused reports and enriched 1-on-1s, we empower you to build high-performing teams that thrive.
                </p>
              </div>
            </div>
            <div
              class="card min-h-[14rem] cursor-pointer"
            >
              <div
                class="card-body relative text-white text-center"
              >
                <div
                  class="absolute bg-green top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
                />
                <h2
                  class="font-semibold text-lg whitespace-nowrap"
                >
                  Employee Development
                </h2>
                <p
                  class="text-base mt-[8%]"
                >
                  Nurture and keep your talent flourishing through personalized, continuous development that's not only easy to measure but also scalable for sustained growth.
                </p>
              </div>
            </div>
          </div>
        </section>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex flex-col gap-y-16 mt-4"
    >
      <div
        class="mx-4 md:mx-8 flex justify-between items-center"
      >
        <a
          class="font-[1000] text-darkPurple text-3xl bg-transparent border-0"
          href="/"
        >
          TG
        </a>
        <div
          class="flex gap-x-4"
        >
          <a
            class="btn btn-md rounded-full capitalize font-normal text-white text-sm md:text-base btn-accent btn-sm md:btn-md px-4 md:px-8"
            data-testid="Sign In-button"
            href="/sign-in"
            id="Sign In-button"
          >
            Sign In
          </a>
          <a
            class="btn btn-md rounded-full capitalize font-normal text-white text-sm md:text-base btn-primary btn-sm md:btn-md px-4 md:px-8"
            data-testid="Sign Up-button"
            href="/sign-up"
            id="Sign Up-button"
          >
            Sign Up
          </a>
        </div>
      </div>
      <section
        class="relative grid grid-cols-1 sm:gap-x-20 sm:grid-cols-2 place-items-center px-4 md:px-8 md:py-20"
      >
        <div
          class="absolute w-full h-full bg-lightPurple opacity-20 z-[-1]"
        />
        <div
          class="flex flex-col gap-y-8"
        >
          <h1
            class="font-semibold text-center md:text-start text-2xl md:text-3xl lg:text-6xl text-black opacity-90"
          >
            Thrive Together for a Vibrant Workplace Culture
          </h1>
          <p
            class="text-sm md:text-base text-center md:text-start"
          >
            Equip your organization with essential tools and insights for employee engagement, performance, and growth.
          </p>
          <div
            class="flex justify-between mx-8 md:mx-0 gap-x-8"
          >
            <a
              class="btn btn-md rounded-full capitalize font-normal text-white text-sm md:text-base btn-secondary flex-grow"
              data-testid="join-now-button"
              href="/sign-up"
              id="join-now-button"
            >
              Join Now
            </a>
            <button
              class="btn btn-md rounded-full capitalize font-normal text-white text-sm md:text-base btn-accent flex-grow"
              data-testid="our-goals-button"
              id="our-goals-button"
            >
              Our goals
            </button>
          </div>
        </div>
        <div
          class="relative min-h-[20rem] w-full h-full col-start-2 flex justify-center my-[8%]"
        >
          <img
            alt="Mindset"
            class="object-contain h-full absolute"
            src="/mindset.svg"
          />
        </div>
      </section>
      <section
        class="px-4 md:px-8 flex flex-col gap-y-16"
      >
        <h1
          class="font-semibold text-2xl md:text-3xl lg:text-4xl"
        >
          How does it work?
        </h1>
        <div
          class="flex flex-col lg:flex-row gap-y-4 md:gap-x-2 lg:gap-x-4"
        >
          <div
            class="card lg:w-1/3 min-h-[14rem] cursor-pointer group"
          >
            <div
              class="card-body relative text-white text-center lg:group-hover:translate-y-4 transition ease-in-out"
            >
              <div
                class="absolute bg-lightPurple top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
              />
              <h2
                class="font-semibold text-lg md:text-2xl whitespace-nowrap"
              >
                Create an Organization
              </h2>
              <p
                class="text-sm md:text-base mt-[8%]"
              >
                Empower your team to navigate corporate challenges.
              </p>
            </div>
          </div>
          <div
            class="card lg:w-1/3 min-h-[14rem] cursor-pointer group"
          >
            <div
              class="card-body relative text-white text-center lg:group-hover:translate-y-4 transition ease-in-out"
            >
              <div
                class="absolute bg-darkPurple top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
              />
              <h2
                class="font-semibold text-lg md:text-2xl whitespace-nowrap"
              >
                Take a Survey
              </h2>
              <p
                class="text-sm md:text-base mt-[8%]"
              >
                Share your thoughts and make your voice count!
              </p>
            </div>
          </div>
          <div
            class="card lg:w-1/3 min-h-[14rem] cursor-pointer group"
          >
            <div
              class="card-body relative text-white text-center lg:group-hover:translate-y-4 transition ease-in-out"
            >
              <div
                class="absolute bg-green top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
              />
              <h2
                class="font-semibold text-lg md:text-2xl whitespace-nowrap"
              >
                Receive Feedback
              </h2>
              <p
                class="text-sm md:text-base mt-[8%]"
              >
                Keep your team engaged with personalized notifications and insights.
              </p>
            </div>
          </div>
        </div>
      </section>
      <section
        class="px-4 md:px-8 flex flex-col gap-y-8"
        id="goals"
      >
        <h1
          class="font-semibold text-2xl md:text-3xl lg:text-4xl pt-4 md:pt-8"
        >
          Why Us?
        </h1>
        <div
          class="md:grid md:grid-cols-2 grid-rows-1 place-items-center gap-x-8 group hidden"
        >
          <div
            class="flex flex-col gap-y-4"
          >
            <h2
              class="text-darkPurple font-semibold text-2xl md:text-3xl group-hover:text-lightPurple transition ease-in-out"
            >
              Employee Engagement
            </h2>
            <p
              class="text-base md:text-lg"
            >
              Open up the channels and lend an ear to your incredible team. By actively listening to your employees, you're not just making decisionsâ€”you're orchestrating a symphony of collaboration, paving the way for meaningful change within your organization.
            </p>
          </div>
          <div
            class="relative min-h-[20rem] w-full h-full flex justify-center my-[4%]"
          >
            <img
              alt="illustrations"
              class="object-contain h-full absolute"
              src="/engineer_team.svg"
            />
          </div>
        </div>
        <div
          class="md:grid md:grid-cols-2 grid-rows-1 place-items-center gap-x-8 group hidden"
        >
          <div
            class="relative min-h-[20rem] w-full h-full flex justify-center my-[4%]"
          >
            <img
              alt="Illustrations"
              class="object-contain h-full absolute"
              src="/pie_graph.svg"
            />
          </div>
          <div
            class="flex flex-col gap-y-4"
          >
            <h2
              class="text-darkPurple font-semibold text-2xl md:text-3xl group-hover:text-lightPurple transition ease-in-out"
            >
              Performance Management
            </h2>
            <p
              class="text-base md:text-lg"
            >
              Elevate your teams to new heights by fostering ongoing conversations. From anytime feedback and goal tracking to development-focused reports and enriched 1-on-1s, we empower you to build high-performing teams that thrive.
            </p>
          </div>
        </div>
        <div
          class="md:grid md:grid-cols-2 grid-rows-1 place-items-center gap-x-8 group hidden"
        >
          <div
            class="flex flex-col gap-y-4"
          >
            <h2
              class="text-darkPurple font-semibold text-2xl md:text-3xl group-hover:text-lightPurple transition ease-in-out"
            >
              Employee Development
            </h2>
            <p
              class="text-base md:text-lg"
            >
              Nurture and keep your talent flourishing through personalized, continuous development that's not only easy to measure but also scalable for sustained growth.
            </p>
          </div>
          <div
            class="relative min-h-[20rem] w-full h-full flex justify-center my-[4%]"
          >
            <img
              alt="illustrations"
              class="object-contain h-full absolute"
              src="/report.svg"
            />
          </div>
        </div>
        <div
          class="flex flex-col gap-y-4 md:hidden"
        >
          <div
            class="card min-h-[14rem] cursor-pointer"
          >
            <div
              class="card-body relative text-white text-center"
            >
              <div
                class="absolute bg-lightPurple top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
              />
              <h2
                class="font-semibold text-lg whitespace-nowrap"
              >
                Employee Engagement
              </h2>
              <p
                class="text-base mt-[8%]"
              >
                Open up the channels and lend an ear to your incredible team. By actively listening to your employees, you're not just making decisionsâ€”you're orchestrating a symphony of collaboration, paving the way for meaningful change within your organization.
              </p>
            </div>
          </div>
          <div
            class="card min-h-[14rem] cursor-pointer"
          >
            <div
              class="card-body relative text-white text-center"
            >
              <div
                class="absolute bg-darkPurple top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
              />
              <h2
                class="font-semibold text-lg whitespace-nowrap"
              >
                Performance Management
              </h2>
              <p
                class="text-base mt-[8%]"
              >
                Elevate your teams to new heights by fostering ongoing conversations. From anytime feedback and goal tracking to development-focused reports and enriched 1-on-1s, we empower you to build high-performing teams that thrive.
              </p>
            </div>
          </div>
          <div
            class="card min-h-[14rem] cursor-pointer"
          >
            <div
              class="card-body relative text-white text-center"
            >
              <div
                class="absolute bg-green top-0 left-0 w-full h-full z-[-1] opacity-[0.90]"
              />
              <h2
                class="font-semibold text-lg whitespace-nowrap"
              >
                Employee Development
              </h2>
              <p
                class="text-base mt-[8%]"
              >
                Nurture and keep your talent flourishing through personalized, continuous development that's not only easy to measure but also scalable for sustained growth.
              </p>
            </div>
          </div>
        </div>
      </section>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
